### PR TITLE
Fix vulnerabilities

### PR DESCRIPTION
**Pull request by naflcosta**
Upgrade all dev-dependencies (and respective configs) to latest versions, fixing vulnerabilities; logic.spec unit-test fix.

### CODE REVIEW

**Comment by naflcosta** [comment]
@FranzSkuffka and @Ryuno-Ki  can you please review? Thanks.

**Comment by Ryuno-Ki** [comment]
Can't do, since I neither work with babel nor webpack.

I couldn't find any glaring issues, though

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "plugins": [
+    // we'll be using https://www.npmjs.com/package/babel-plugin-add-module-exports
+    "add-module-exports"
+  ],
+  "presets": [
+    [
+      "@babel/preset-env",
```
Comment (FranzSkuffka) [inline .babelrc]: good.

File: package.json [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mercedesbenzio/iframe-hash-manager.git"
+  },
   "scripts": {
     "test": "webpack --config webpack.tests.js && ava src/*.spec.js",
     "build": "webpack",
     "watch-build": "webpack --watch",
-    "prepublish": "./npm run test && ./npm run build"
+    "prepublishOnly": "./npm run test && ./npm run build"
```
Comment (FranzSkuffka) [inline package.json]: nice

**Pull request merged**